emissionscheck_alb/baseline_model.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import numpy as np
+
+
+def run_baseline_model(features_train, features_test, target_train, target_test, model, random_state=None, class_weight=None, max_fpr=None):
+    """
+    Run logistic regression or SVC  
+    
+    """
+    from sklearn.pipeline import make_pipeline
+    from sklearn.preprocessing import MinMaxScaler
+    from sklearn.metrics import roc_auc_score, plot_confusion_matrix, classification_report
+    from sklearn.model_selection import cross_val_score
+
+    if model == "logreg":
+
+        from sklearn.linear_model import LogisticRegression
```

Comment:
import all your packages at the top of your script
```suggestion
```

Reply:
Done